FirebasePhoneAuthUI/FUIPrivacyAndTermsOfServiceView+PhoneAuth.h

```diff
@@ -0,0 +1,31 @@
+//
+//  Copyright (c) 2016 Google Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#import "FUIPrivacyAndTermsOfServiceView.h"
```

Comment:
Looks like Travis is failing on this import statement, though I'm not sure why. 

```
❌  /Users/travis/build/firebase/FirebaseUI-iOS/FirebasePhoneAuthUI/FUIPrivacyAndTermsOfServiceView+PhoneAuth.h:17:9: 'FUIPrivacyAndTermsOfServiceView.h' file not found
#import "FUIPrivacyAndTermsOfServiceView.h"
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
should be resolved.